### ISSUE TITLE
is it required to use proxy?

### ISSUE THREAD

**Issue by anantsparrow** [reacherhq/check-if-email-exists#1394]
### From where did you run check-if-email-exists?

hetzner

### What happened?

Hi there, i am unable to host this api to my server

**Comment by amaury1729** [comment]
No, a proxy is not required. Make sure port 25 is open on your server. What error do you have?

**Comment by amaury1729** [comment]
This looks like a port 25 issue. Make sure you have it open. See [guide](https://help.reacher.email/self-host-guide#5cd3264fca29473e8d68664587e7d410).

**Comment by anantsparrow** [comment]
{
    "input": "vivek821038@gmail.com",
    "is_reachable": "unknown",
    "misc": {
        "is_disposable": false,
        "is_role_account": false
    },
    "mx": {
        "accepts_mail": true,
        "records": [
            "alt1.gmail-smtp-in.l.google.com.",
            "gmail-smtp-in.l.google.com.",
            "alt3.gmail-smtp-in.l.google.com.",
            "alt2.gmail-smtp-in.l.google.com.",
            "alt4.gmail-smtp-in.l.google.com."
        ]
    },
    "smtp": {
        "error": {
            "type": "SocksError",
            "message": "i/o error: could not resolve address (\"\", 587)"
        }
    },
    "syntax": {
        "address": "vivek821038@gmail.com",
        "domain": "gmail.com",
        "is_valid_syntax": true,
        "username": "vivek821038"
    }


getting this socks error, also my server has port 25 open sir
